packages/frontend/README.md

```diff
@@ -7,3 +7,12 @@ In order to effectively contribute to the frontend, you should read the [contrib
 To start developing with the frontend, you should run the `npm run start:dev` script at the project root. Make sure you install dependencies (i.e. `yarn`) at the project root first.
 
 For prebuilt UI components, you can use the NextUI package. More custom components can be build using TailwindCSS combined with the classnames package.
+
+# Running with Docker
```

Comment:
Running tests in docker is helpful when you're trying to do non mocked db things. Maybe add a yarn command `test:docker` that does this